ccd-client/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=uk.gov.hmcts.cmc.ccd.client.SaveCaseService
```

Comment:
shouldn't this already be scanned as its in our repo?

Reply:
I think no, as base package does not match with the claimstore.